pytest_cases/filters.py

```diff
@@ -0,0 +1,57 @@
+import re
+
+from .case_info import CaseInfo
+
+
+class CaseFilter:
```

Comment:
py2 compat just in case

```suggestion
class CaseFilter(object):
```